packages/build/src/lib/config/create-config.ts

```diff
@@ -16,6 +16,19 @@ export function createConfig(
   const normalizedOptions = normalizeOptions(options);
   const isProduction = process.env['NODE_ENV'] === 'production';
 
+  let lessPathOptions: { paths?: string[] } = {};
+  let sassPathOptions: { includePaths?: string[] } = {};
+  if (options.stylePreprocessorOptions?.includePaths) {
+    if (options.stylePreprocessorOptions.includePaths.length > 0) {
+      lessPathOptions = {
+        paths: options.stylePreprocessorOptions.includePaths,
+      };
+      sassPathOptions = {
+        includePaths: options.stylePreprocessorOptions.includePaths,
+      };
+    }
+  }
```

Comment:
```suggestion
  function getPathOptions(includePaths?: string[]) {
  if (!includePaths || includePaths.length === 0) {
    return { less: {}, sass: {} };
  }

  return {
    less: { paths: includePaths },
    sass: { includePaths },
  };
}

const { less: lessPathOptions, sass: sassPathOptions } = getPathOptions(
  options.stylePreprocessorOptions?.includePaths
);
```

I suggest to avoid mutations and go with a functional approach. (we get lint rules for this kind of feedback)